server/src/main/java/org/apache/iotdb/db/metadata/MTree.java

```diff
@@ -222,7 +223,7 @@ MeasurementMNode createTimeseries(
     if (nodeNames.length <= 2 || !nodeNames[0].equals(root.getName())) {
       throw new IllegalPathException(path.getFullPath());
     }
-    checkTimeseries(path);
+    LogicalGenerator.checkCreateTimeseriesGrammar(path.getFullPath());
```

Comment:
hi ,  I have a little question. Can we perform verification when the SQL (string) is converted into a plan instead of during metadata operations?

Reply:
Because there are many situations that lead to the creation of `CreateTimesPlan`, but in the end, this method will be called, so I wrote here to prevent some situations from being missed.